spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/server/web/reactive/InstancesProxyControllerIntegrationTest.java

```diff
@@ -33,15 +32,12 @@
 	@Nullable
 	private static ConfigurableApplicationContext context;
 
-	@BeforeClass
-	public static void setUpContext() {
+	@Before
+	public void setUpClient() {
```

Comment:
Failing: https://travis-ci.org/github/codecentric/spring-boot-admin/builds/680925187
Working: https://travis-ci.org/github/codecentric/spring-boot-admin/builds/680931402

The difference is now that for every test a own Spring Boot application context is bootstrapped. 
I'll further analyze what exactly cause the problem